src/evm_context.rs

```diff
@@ -168,6 +189,26 @@ pub fn mload(index: &String, ssa: &mut SSATracker) -> String {
     format!("(concat {})", arguments)
 }
 
+pub fn keccak256(offset: &String, length: &String, ssa: &mut SSATracker) -> String {
+    let offset_0 = mload(offset, ssa);
+    let offset_32 = mload(&format!("(bvadd {} #x{:064X})", offset, 0x20), ssa);
+    smt::ite(
+        &smt::eq(length, &0x20),
+        &format!("({} {})", keccak256_32(), offset_0),
+        &smt::ite(
+            &smt::eq(length, &0x40),
+            &format!("({} {} {})", keccak256_32(), offset_0, offset_32),
```

Comment:
```suggestion
            &format!("({} {} {})", keccak256_64(), offset_0, offset_32),
```
:)

Reply:
Why did this not result in an smt parser error?

Reply:
good q, can you check that the tests are covering this?